prettier/prettier-check.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prettier-check-from-gitrepo
+spec:
+  workspaces:
+  - name: source
+    mountPath: /source
+  params:
+  - name: subdirectory
+    type: string
+    description: Subdirectory inside "source" workspace that contains the target files.
+    default: "."
+  - name: filepath
+    type: string
+    description: path of file of directory inside subdirectory that contains the target file(s).
```

Comment:
Did you mean "file or directory" instead of "file of directory" here?

Reply:
Yeah, sorry it is a typo